### DISCUSSION TITLE
Import/ Export Realm like Keycloak

### DISCUSSION THREAD

**Discussion by MarvinHofmann** [OpenIdentityPlatform/OpenAM#734] [Q&A]
Hello everyone,

is it possible to import/export realms like with keycloak by a button in the Admin UI or even better in a Dockerfile?

**Comment by vharseko** [comment]
[Backing Up and Restoring OpenAM Configurations](https://doc.openidentityplatform.org/openam/admin-guide/#chap-backup-restore)
